src/assignment/helpers/delete-assignment.ts

```diff
@@ -39,6 +41,9 @@ export async function deleteAssignment(
 				object: assignmentId,
 				object_type: 'assignment',
 				action: 'delete',
+				resource: {
+					education_level: String(assignment?.education_level_id),
```

Comment:
why are these casts to String needed?
This will also cause the value to be the string 'undefined' if it is undefined
Why not just leave it as what it was without the cast?

same remark for all events

Reply:
`resource` is typed as `Record<string, string | string[] | boolean | number | Avo.Search.DateRange>`, `undefined` is none of ` string | string[] | boolean | number | Avo.Search.DateRange`.